client/src/components/Navbar.js

```diff
@@ -1,116 +1,79 @@
-import React, {useState} from "react";
-import { Link, withRouter } from "react-router-dom";
-
+import React from 'react';
+import { NavLink, withRouter } from 'react-router-dom';
 import useAuth from '../hooks/useAuth';
-import {authLevelRedirect} from '../utils/authUtils'
+import HflaImg from '../svg/hflalogo.svg';
+import { Box, Button, Grid } from '@mui/material';
+import { styled } from '@mui/system';
+import theme from '../theme';
 
-import "../sass/Navbar.scss";
-//hooks
-//function -> class
 const Navbar = (props) => {
+  // check user accessLevel and adjust link accordingly
+  const { auth } = useAuth();
+ 
+  // Styles that may need to be included in the theme.
```

Comment:
You can remove this comment. The `Button` element styling that you are using for the navbar is specific to only the navbar file